packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -285,6 +300,14 @@ IFluidDataStoreChannel, IFluidDataStoreRuntime, IFluidHandleContext {
         if (existing) {
             this.deferredAttached.resolve();
         }
+
+        if (initializeEntrypoint) {
+            this.handle = new FluidObjectHandle<FluidObject>(
```

Comment:
i don't know if i did it in my draft, but something we could consider is to get the handle from the object itself here, by calling request in the absence of initializeEntrypoint. i wouldn't do it in this change, but it would allow the handle to always be available, even if the implementor has yet to provide an initializeEntrypoint function